lib/client/lambda-server.ts

```diff
@@ -107,7 +121,7 @@ export class LambdaServer {
                 console.log(JSON.stringify(bodyJSON, null, 2));
             }
 
-            const handlerFunction = this.functionName ? this.functionName : "handler";
+            handlerFunction = handlerFunction != null ? handlerFunction : this.functionName ? this.functionName : "handler";
```

Comment:
I recommend going with 
`handlerFunction = handlerFunction ? handlerFunction : ...`
To include undefined apart from null, right now it's undefined if request.url is "/"